### PR TITLE
#finalView: I got stuck here :)

### PR DESCRIPTION
**Pull request by alqayadhi**
as it shown below the finalViewResult comes when we answer the first question:
image
.

### CODE REVIEW

File: src/pages/questionPage.js [hunk 1]
```diff
@@ -46,10 +47,12 @@ export const initQuestionPage = () => {
 const nextQuestion = () => {
   quizData.currentQuestionIndex = quizData.currentQuestionIndex + 1;
   initQuestionPage();
+  finalSummaryPage();
```
Comment (arkaitzgarro) [inline src/pages/questionPage.js:50]: You are calling the function immediately, you need to add a condition to only call the function after the last question.

File: index.html [hunk 2]
```diff
@@ -1,23 +0,0 @@
-<!DOCTYPE html>
```
Comment (arkaitzgarro) [inline index.html:1]: Why did you remove the whole index file?

File: src/views/finalView.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import { quizData } from '../data.js';
 
 export const createFinalSummaryElement = () => {
-  const { finalScore, questions } = quizData;
+  const { currentQuestionIndex, questions } = quizData;
 
   const element = document.createElement('div');
   element.innerHTML = `
-      <h1 class='final'>Congratulations ! You answered ${finalScore} correct from  ${questions.length} questions!</h1>`;
+      <h1 class='final'>Congratulations ! You answered ${currentQuestionIndex} correct from  ${questions.length} questions!</h1>`;
```
Comment (arkaitzgarro) [inline src/views/finalView.js:8]: We want to show the final score, not the index of the question.

File: src/pages/questionPage.js (hunk 1 above)
Comment (alqayadhi) [reply src/pages/questionPage.js:50]: I have added the conditional function:
if (quizData.currentQuestionIndex === quizData.questions.length - 1){
  finalSummaryPage();
  return;
  }

File: src/views/finalView.js (hunk 3 above)
Comment (alqayadhi) [reply src/views/finalView.js:8]: okay

File: index.html (hunk 2 above)
Comment (alqayadhi) [reply index.html:1]: accidentally :(

**Comment by arkaitzgarro** [comment]
Implemented in #36
